src/main/java/com/foo/gosucatcher/domain/estimate/application/dto/response/MemberEstimateResponse.java

```diff
@@ -16,11 +17,12 @@ public record MemberEstimateResponse(
 	public static MemberEstimateResponse from(MemberEstimate memberEstimate) {
 
 		if (memberEstimate == null) {
-			return new MemberEstimateResponse(null, null, null, null, null, null);
+			return new MemberEstimateResponse(null, null, null, null, null, null, null);
```

Comment:
```suggestion
			return new MemberEstimateResponse();
```
차라리 매개변수 없는 생성자 하나 만드는거 어떤가요? 다 null들어가는데 이렇게 초기화하는게 어색해서요